queries/javascript/highlights.scm

```diff
@@ -184,6 +184,7 @@
 
 (binary_expression "/" @operator)
 (ternary_expression ["?" ":"] @operator)
+(unary_expression) @operator
```

Comment:
This is incorrect. The unary expression node covers the indentifier argument as well. This should be `(unary_expression "!") @operator`, matching the anonymous node.